### PR TITLE
Add users rating

### PR DESCRIPTION
**Pull request by AmayAgrawal**
Right now I have implemented it but not stored anything.It first authenticates the user and then can take a rating and feedback about all sessions.Only thing what you have to do in it is add your client Id in index.html and index1.html where it is written your client Id here.If you think it is alright then let me know I will add it to database that rating and feedback.Also u feel something is not right then also let me know.

### CODE REVIEW

**Comment by AmayAgrawal** [comment]
Can u please elaborate on the above notes.I didn't get what you are trying to say in all the above notes.

**Comment by leto** [comment]
@AmayAgrawal the @houndci-bot account is an automated robot that looks for possible errors/etc. The DateUtils not being defined sounds important. The "line is too long" is probably not a big deal.

**Comment by AmayAgrawal** [comment]
I exactly don't know the DateUtils function.Can u please suggest me or explain me somewhat about its implementation.

File: app/components/sessions/SessionsController.js [hunk 1]
```diff
@@ -77,13 +78,25 @@ sessionsModule.controller('SessionsController',
...
                     controller: 'SessionDialogController',
                     templateUrl: 'app/components/sessions/sessiondialog.html',
                     parent: angular.element(document.body),
                     targetEvent: event,
 
                 });
+               }
+               else{
+                $mdDialog.show({
+                    controller: 'SessionLoginDialogController',
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406561178/issues/source?bid=3233857&fileBranchId=3233876#l93)

File: app/components/sessions/SessionsController.js [hunk 2]
```diff
@@ -104,4 +117,21 @@ sessionsModule.controller('SessionDialogController', ['$mdDialog',
 
             return start + ' - ' + end;
         };
-    }]);
\ No newline at end of file
+    }]);
+
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406561178/issues/source?bid=3233857&fileBranchId=3233876#l123)

File: app/components/sessions/SessionsController.js [hunk 3]
```diff
@@ -77,13 +78,25 @@ sessionsModule.controller('SessionsController',
...
                 $mdDialog.show({
                     controller: 'SessionDialogController',
                     templateUrl: 'app/components/sessions/sessiondialog.html',
                     parent: angular.element(document.body),
                     targetEvent: event,
 
                 });
+               }
+               else{
+                $mdDialog.show({
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: [Strings must use doublequote.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406561178/issues/source?bid=3233857&fileBranchId=3233876#l92)

**Pull request closed without merging**